src/main/kotlin/racingcar/domain/Name.kt

```diff
@@ -0,0 +1,14 @@
+package racingcar.domain
+
+@JvmInline
+value class Name(private val name: String) {
+    fun getName() = name
+
+    init {
+        require(name.length <= MAXIMUM_NAME_LENGTH) { "이름은 10자 이하여야 합니다." }
```

Comment:
```suggestion
        require(name.length <= MAXIMUM_NAME_LENGTH) { "이름은 ${MAXIMUM_NAME_LENGTH}자 이하여야 합니다." }
```
이렇게 작성하면 요구사항 변경에 따른 코드 변경을 최소화할 수 있어요.